docs/mindsdb-docs/docs/sql/tutorials/predicting_house_price_tutorial.md

```diff
@@ -0,0 +1,110 @@
+## Predicting house price
+In this tutorial you will learn how to predict house price on basis of different parameter like condition,bedrooms,bathrooms,area etc. We will be using mysql database for this purpose and kc_house_data as dataset.
+The process will be like:
+
+ 1. Cleaning data and deleting unnecessary columns.
+ 2. Starting MySQL main server to create databases and tables.
+ 3. Starting mindsdb server to access gui and its database.
+ 5. Starting another MySQL server on port 47735 with username mindsdb
+ 7. Predicting using our predictor.
+
+These are our 5 step to add ML prediction model layer on any mysql database. 
+So let get started.
+
+## Cleaning data:
+Simply install database and clean it using excel ,your data should look like this:
+![excle_img](https://user-images.githubusercontent.com/59201258/141510855-b8e2325a-361c-44c4-9fd8-8adfb6729602.PNG)
```

Comment:
I think it would be good to exclude this since the dataset looks ok, no need for transformation or data cleaning?

Reply:
ya because database include some columns that shouldn't be there